public/index.html

```diff
@@ -1,18 +1,19 @@
-<!DOCTYPE html>
-
-<html lang="en">
-
-<head>
-
-    <title>Pieces Demo Todo List</title>
-</head>
-
-<body>
-<noscript>
-    You need JS to run this application!
-</noscript>
-<div id="root"></div>
-</body>
-
-
-</html>
+<html><head>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+        <style type="text/css">
+            body{font-family: "poppins",serif;background-color: #dad9d9;color:#0b0c0c}
+        </style>
+     </head>
+     <body>
+        <img src="c:\Users\Girish\Desktop\website\wave.png">
```

Comment:
This image won't be visible to us, since it points to a local file. This should either be inside an assets/static folder containing the image.

Reply:
I am aware of this issue, that's why I asked @jordan-pieces for access to the assets folder, I haven't got that yet